packages/exhibitor/extra/dcos-exhibitor.service

```diff
@@ -10,6 +10,9 @@ RestartSec=5
 # Run in new mount namespace to create custom resolv.conf.
 MountFlags=private
 RuntimeDirectory=dcos_exhibitor
+# On some systems /tmp is mounted as noexec. Make zookeeper write its JNA
+# libraries to a path we control instead. See DCOS-11056
+Environment=SERVER_JVMFLAGS=-Djna.tmpdir=/var/lib/dcos/exhibitor/tmp
```

Comment:
I'm would be happy to do that. Have a quick look at https://github.com/dcos/dcos/blob/master/gen/dcos-config.yaml#L747. 

There are several options for moving this env var to that section, none of them are clean. The cleanest is probably extracting 

```
  - path: /etc/exhibitor
    content: |
```
from the start of the individual cases:

```
  - path: /etc/exhibitor
    content: |
      SERVER_JVMFLAGS=...
{% switch exhibitor_storage_backend %}
{% case "agent_only_group_no_exhibitor" %}
...
{% endswitch %}
```
But that doesn't quite work as the  `agent_only_group_no_exhibitor` case is mixed in there, too.

I don't see any way to add this env var to that file that doesn't involve copy-pasting it to all the different cases.

In general I would also prefer setting it in this file, but that file is being generated in a weird way.

What do you think?